lib/mongoose_push.ex

```diff
@@ -86,6 +86,8 @@ defmodule MongoosePush do
         :timer.tc(module, :push, [notification, device_id, pool, opts])
       end
 
+    emit_telemetry_event(time, push_result)
+
     push_result
     |> Metrics.update(:spiral, [:push, service, mode])
```

Comment:
Metrics updates in elixometer should not longer be visible anywhere in core logic. Those calls should be made in telemetry handlers.